### DISCUSSION TITLE
Members of our organization should not edit my post

### DISCUSSION THREAD

**Discussion by shiva-sandupatla** [community/community#3205] [Discussions]
I have a repo which is in my organization. I am the owner of that organization and my organization member can able to edit my post which is posted in that repo, It is better if that member cannot have access to edit or pin/unpin or lock/unlock my post. It will be better.

Hoping with a positive response

Thanks, @Shiva-sandupatla

**Comment by PranamBhat** [comment]
Hi @shiva-sandupatla 

This is very genuine feedback. Thank you for bringing this one.

@evi-liu @candyho Please have a look here!

**Reply by shiva-sandupatla** [reply]
@PranamBhat Thanks for your response, hope this feature request will be implemented.

Thanks, shiva
